src/test/java/org/opensearch/security/authtoken/jwt/JwtVendorTest.java

```diff
@@ -42,17 +44,20 @@ public void testCreateJwkFromSettingsWithoutSigningKey() throws Exception{
     }
 
```

Comment:
Maybe add a test where roles is empty or where we test incorrect keys?

Reply:
done~